src/06-sort.js

```diff
@@ -23,7 +25,9 @@ function sortByRuntimeAscending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortByArtistNameDescending(songs) {}
+function sortByArtistNameDescending(songs) {
+ return songs.sort((a,b) => {return b.artist.toUpperCase().localeCompare(a.artist.toUpperCase())})
```

Comment:
That's one way to do it! Another approach is using the attached logic to create conditionals that are more in line with how sort works
![Screenshot 2022-11-22 at 11 10 55 AM](https://user-images.githubusercontent.com/71154265/203365416-36134d88-7803-4859-ba4b-5d71c15e4e10.png)